app/templates/suppliers_details.html

```diff
@@ -55,14 +55,26 @@
         <dl class="visually-hidden">
           <dt>Capabilities</dt>
         </dl>
-        {% for category in supplier_categories|sort() %}
+        {% for category in supplier["categories"]|sort() %}
         <dd>
           <a href="#" class="disabled">{{ category }}</a>
         </dd>
         {% endfor %}
       </footer>
     </article>
   </li>
+  {% if supplier["caseStudies"]|count > 0 %}
+  <li>
+    <p>Case Studies</p>
+    <article>
+      {% for casestudy in supplier["caseStudies"] %}
+        {% with component = render_component('bundles/CaseStudy/CaseStudyViewWidget.js',{ "casestudy": casestudy }) %}
```

Comment:
won't this display the full case study?

don't we just want a list of case studies with titles that links to the view page?

(I would just do in jinja for now)